app/forms/signals_telegram_bot_registration_form.rb

```diff
@@ -24,13 +23,36 @@ def user
     @user ||= User.where("(token_sale->>'telegram_username') ILIKE ?", telegram_username).first
   end
 
+  def validate_user_access
+    signals_access_override = user.token_sale.fetch('signals_access_override', nil)
+
+    # Handle `signals_access_override` values
+    unless signals_access_override.nil?
+      if signals_access_override == true
```

Comment:
I actually wanted to avoid just truthy values and instead throw an error if it is not exactly boolean. I think it just helps avoid any `"false"` strings from accidentally returning `true`